### ISSUE TITLE
Transit Accessibilities seem to be off

### ISSUE THREAD

**Issue by moeckel** [msmobility/silo#273]
When I plot auto accessibility, the pattern looks very convincing and intuitive. Plotting the transit accessibility seem to provide a wrong pattern. The pattern should be smooth and not as fragmented. The picture should be similar to auto accessibilities, but hopefully be visually affected by transit lines. I got both auto and transit accessibility out of the spatial result file in the based year in the base scenario.

Auto Accessibility (Quantile Scale)
![potentialAutoAccessibility](https://user-images.githubusercontent.com/7513703/70478087-88e31e80-1ada-11ea-8130-474a87464320.png)

Transit Accessibility (Quantile Scale)
![potentialTransitAccessibility](https://user-images.githubusercontent.com/7513703/70478201-c5af1580-1ada-11ea-8c91-b0041818160f.png)

**Comment by nkuehnel** [comment]
which setup did you use? skim or schedule?

**Comment by moeckel** [comment]
I used the base scenario from Jerry. I assume it is skim-based, at least the properties file does not specify anything else. But that would be helpful to confirm, good point. 
[inputProperties.txt](https://github.com/msmobility/silo/files/3942223/inputProperties.txt)

**Comment by nkuehnel** [comment]
I think this has been fixed
